jobs/curator/templates/bin/curator_ctl

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+set -u
+set -x
+
+CRON_SCHEDULE="*/2 * * * * /var/vcap/jobs/curator/bin/purge_logs"
+CRON_STATUS=`pgrep cron | wc -l`
+
+source /var/vcap/jobs/curator/helpers/ctl_setup.sh 'curator'
+
+case $1 in
+  start)
+    if [ $CRON_STATUS -eq 0 ]; then
+      /usr/sbin/cron start
+    fi
+
+    if [ -n "$(crontab -l)" ]; then
+      crontab -l > /tmp/crontab_buffer
+    fi
+
+    if [ -z "$(crontab -l | grep 'purge_logs')" ] ; then
+      echo "$CRON_SCHEDULE" >> /tmp/crontab_buffer
+    fi
+
+    crontab < /tmp/crontab_buffer
```

Comment:
This whole approach is kind of fragile with the listing/grepping/appending on terms. Did you have a look at how the [archiver](https://github.com/logsearch/logsearch-boshrelease/blob/develop/jobs/archiver/templates/bin/logstash.process.erb#L49) installs a cron task - it's much simpler and, as far as I know, hasn't been a problem for anybody. If you did have problems, I'd be interested to hear and fix (and in helping come up with a less fragile approach than this one).